src/qml/EntryComponentList.qml

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of system-settings
+ *
+ * Copyright (C) 2013-2016 Canonical Ltd.
+ * Copyright (C) 2020 Ubports Foundation <developers@ubports.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 3, as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranties of
+ * MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.4
+import SystemSettings.ListItems 1.0 as SettingsListItems
```

Comment:
ListItems is [deprecated](https://api-docs.ubports.com/sdk/apps/qml/Ubuntu.Components.ListItems/Base.html#detailed-description). Please, don't use them in new code, instead use ListItem

Reply:
No, these are not the UITK's listitems, but some special classes used in SystemSettings (note the import line). And internally they use the new ListItem.

Reply:
Oh, sorry.